docs/checklists/tandem.md

```diff
@@ -242,3 +242,17 @@ At the moment, we only process data from the most recent pump shut-down event, i
   - `[ ]` activity/exercise
   - `[ ]` food (e.g., from a food database built into the pump)
   - `[ ]` notes/other events
+
+### Tidepool ingestion API
+
+Choose one of the following:
+
+  - `[x]` legacy "jellyfish" ingestion API
+  - `[ ]` platform ingestion API
+
+### Known implementation issues/TODOs
+
+*Use this space to describe device-specific known issues or implementation TODOs **not** contained in the above datatype-specific sections.*
+
+  - `*[ ]` On first upload, read back to the very beginning of the pump's history to construct a full `pumpSettings` history, allowing for fuller `pumpSettings` history (including profile changes) as well as basal schedule look-up.
+  - `*[ ]` Find a reliable way to identify (and then drop) `timeChange` events occurring after a device shutdown so that BtUTC doesn't have to be limited to records since the last pump shutdown.
```

Comment:
This is also in the checklist already:

> At the moment, we only process data from the most recent pump shut-down event, if any exist. This is because a subtle assumption of BtUTC is that the pump clock is always running. Unfortunately, when a Tandem device is shut down, the clock stops. There are several ways we might try to handle these shut-downs and still do bootstrapping, but we'll need to research and potentially prototype these to figure out the best approach and then implement it.

Maybe merge them into one bullet point?